Loop Simple Calculator.py

```diff
@@ -18,11 +18,7 @@ def simpcalc() :
 		print(subtraction) 
 		
 	elif op == "/" :
-		if num1 or num2 == 0:
-			num1 = int(num1)
-			num2 = int(num2)
-			division = num1 / num2
-		else:
+		if num2 != 0 :
 			division = num1 / num2
 			print(division)
```

Comment:
(num1 or num2 == 0) really means ((num1) or (num2 == 0)) so this will be always true if num1 is true(in case of numbers, that means non-zero) so you couldn't perform any division except for (zero/nonzero) which is not really helpful.